Parsers/ASimNetworkSession/Tests/SonicWall_SonicOS_ASimNetworkSession_DataTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [DvcIpAddr] of type [IP Address]: [""18C24135E8B6""] (Schema:NetworkSession)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 1000 records (100.0%) for field [EventProduct] of type [Enumerated]: [""SonicOS""] (Schema:NetworkSession)"

```

Comment:
Please add this to supported values in EventProduct for NetworkSession: https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv

Reply:
Isn't the product is 'Firewall' instead of 'SonicOS'? If its 'Firewall', please set in parser. no need to update the list as its already a supported value. We should add product name as per vendor documentation.

Reply:
DvcIpAddr has been removed. I've observed it sometimes sent as an IP and other times as a different identifier. To err on the safe side. I've removed the field.

The type of product is a firewall but I set EventProduct to "SonicOS" because that's what our firewall's operating system is called. Going by the examples set by others before me, I figured "SonicOS" was okay. See the example lines below where many could be generalized as "Firewall". If I should change it, please let me know.

Sample from the ASIMTester.csv:
"EventProduct,string,Mandatory,NetworkSession,Enumerated,Fortigate|IOS|ISE|SDP|Vectra Stream|NSGFlow|Fireware|VPC|Azure Defender for IoT|Azure Firewall|M365 Defender for Endpoint|Sysmon|Sysmon for Linux|Windows Firewall|WireData|ZIA Firewall|CDL|PanOS|VMConnection|Meraki|Zeek|Firewall|ASA|Cynerio|SentinelOne|WAF|Firepower|FalconHost|SonicOS,
EventProduct,string,Mandatory,NetworkSession,Enumerated,Fortigate|IOS|ISE|SDP|Vectra Stream|NSGFlow|Fireware|VPC|Azure Defender for IoT|Azure Firewall|M365 Defender for Endpoint|Sysmon|Sysmon for Linux|Windows Firewall|WireData|ZIA Firewall|CDL|PanOS|VMConnection|Meraki|Zeek|Firewall|ASA|Cynerio|SentinelOne|WAF|Firepower|Carbon Black Cloud|SonicOS,"

Reply:
@jaimeesc 
If SonicOS is what is popular with, its fine keep it.
Please update in tester.csv for NetworkSession schema:
EventProduct=> SonicOS
EventVendor=> SonicWall

Reply:
in this PR iteself.

Reply:
I looked for other fields and have the following set:
EventVendor = "SonicWall"
EventProduct = "Firewall"
DvcOs = "SonicOS"
DvcOsVersion = EventProductVersion (aliased EventProductVersion which is renamed from DeviceVersion)
DvcDescription = DeviceProduct